### PR TITLE
Correct a comment and an error

### LINKED ISSUE #342: Issue setting up a node to run on testnet
Copied from @rajkaramchedu:

1) I believe running subtensor node for mainnet is okay, no complaints there. 
2) But whenever a user wants to run the node on testnet, for some reason it keeps going to mainnet, is what they are saying. 
3) It may be as simple as one of the nucleus team members to look at testchain command syntax for both "using source" and "using docker" sections of the doc and help identify what's wrong with those command options. Maybe the bootnode public key is wrong for the testnet? https://docs.bittensor.com/subtensor-nodes/using-source#lite-node-on-testchain
[3:51 PM]
Latest issue the user is facing is from a few mins ago https://discord.com/channels/799672011265015819/1228380578949369947/1230596951381119079. This is all familiar, we've been getting these questions since almost 2+ weeks ago.
[3:53 PM]
This issue resolution alone will reduce a good number of subtensor node related questions in the community.
Doc Issue on this https://github.com/opentensor/developer-docs/issues/188

### PR DESCRIPTION
**Pull request by open-junius**
fixes #342 

## Description
Find out an wrong error type and wrong comment.

## Related Issue(s)

- No

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please describe):

## Breaking Change

If this PR introduces a breaking change, please provide a detailed description of the impact and the migration path for existing applications.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have run `cargo fmt` and `cargo clippy` to ensure my code is formatted and linted correctly
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots (if applicable)

Please include any relevant screenshots or GIFs that demonstrate the changes made.

## Additional Notes

Please provide any additional information or context that may be helpful for reviewers.

### CODE REVIEW

**Review by sam0x17 — APPROVED** [review]

File: pallets/subtensor/src/block_step.rs [hunk 1]
```diff
@@ -517,7 +517,7 @@ impl<T: Config> Pallet<T> {
         }
     }
 
-    // Performs the burn adjustment by multiplying the current difficulty by the ratio ( reg_actual + reg_target / reg_target * reg_target )
+    // Performs the burn adjustment by multiplying the current burn by the ratio ( reg_actual + reg_target / reg_target * reg_target )
     // We use I110F18 to avoid any overflows on u64. Also min_burn and max_burn bound the range.
     //
     pub fn adjust_burn(
```
Comment (gztensor) [inline pallets/subtensor/src/block_step.rs]: Actually, the function name is off: It doesn't adjust anything, just returns updated value. Maybe we should rename it too? Something like `get_upgated_burn` is more accurate.

File: pallets/subtensor/src/block_step.rs [hunk 2]
```diff
@@ -517,7 +517,7 @@ impl<T: Config> Pallet<T> {
         }
     }
 
-    // Performs the burn adjustment by multiplying the current difficulty by the ratio ( reg_actual + reg_target / reg_target * reg_target )
+    // Performs the burn adjustment by multiplying the current burn by the ratio ( reg_actual + reg_target / reg_target * reg_target )
```
Comment (gztensor) [inline pallets/subtensor/src/block_step.rs]: Is updated burn calculated correctly? Why is there "target_registrations_per_interval + target_registrations_per_interval" in the code (addition vs. multiplication)?

File: pallets/subtensor/src/block_step.rs (hunk 1 above)
Comment (open-junius) [reply pallets/subtensor/src/block_step.rs]: yes, the function doesn't update anything. just calculate the burn to be adjusted.

File: pallets/subtensor/src/block_step.rs (hunk 2 above)
Comment (open-junius) [reply pallets/subtensor/src/block_step.rs]: you are right. according to code, it should be addition. will update it also

File: pallets/subtensor/src/block_step.rs (hunk 1 above)
Comment (sam0x17) [reply pallets/subtensor/src/block_step.rs]: yes let's do the rename then, could just be `upgraded_burn`

**Review by sam0x17 — APPROVED** [review]

**Pull request merged**